packages/application-shell/README.md

```diff
@@ -65,13 +65,13 @@ ReactDOM.render(<EntryPoint />, document.getElementById('root'));
 
 ## Props
 
-| Props                      | Type     | Required | Default | Description                                                                                                                                   |
-| -------------------------- | -------- | :------: | ------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
-| `i18n`                     | `object` |    ✅    | -       | An object containing all the translated messages per locale (`{ "en": { "Welcome": "Welcome" }, "de": { "Welcome": "Wilkommen" }}`).          |
-| `configuration`            | `object` |    ✅    | -       | The current `window.app`.                                                                                                                     |
-| `render`                   | `func`   |    ✅    | -       | The function to render the application specific part. This function is executed only when the application specific part needs to be rendered. |
-| `trackingEventWhitelist`   | `object` |    ✅    | -       | An object containing a map of tracking events (_this mapping is required for backwards compatibility, it might be removed in the future_)     |
-| `onRegisterErrorListeners` | `func`   |    ✅    | -       | A callback function to setup global event listeners, called when the `ApplicationShell` is mounted                                            |
+| Props                      | Type               | Required | Default | Description                                                                                                                                                                                                       |
+| -------------------------- | ------------------ | :------: | ------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `applicationMessages`      | `object` or `func` |    ✅    | -       | Either an object containing all the translated messages per locale (`{ "en": { "Welcome": "Welcome" }, "de": { "Welcome": "Wilkommen" }}`), or a function that returns a promise that resolves to such an object. |
```

Comment:
this was outdated

Reply:
I would document how this works. Maybe add a section below about `## Loading i18n messages with code splitting`